dist/aws-sdk-react-native.js

```diff
@@ -33977,6 +33977,11 @@ return /******/ (function(modules) { // webpackBootstrap
 	  initialize: function(list, validate) {
 	    var self = this;
 	    self.values = [].concat(list);
+	    if (Symbol && Symbol.iterator) {
```

Comment:
This change is not required, as this file is auto-generated during release.
Git history https://github.com/aws/aws-sdk-js/commits/master/dist/aws-sdk-react-native.js

Reply:
Removed

Reply:
The changes in dist are still there.
Instead of force-pushing, you can also remove dist changes in a new commit. All PR commits will be squashed into a single commit before merging to master